.gitignore

```diff
@@ -14,6 +14,7 @@
 *.nuget.props
 *.nuget.targets
 NuGet.exe
+nuget.exe
```

Comment:
https://github.com/dotnet/roslyn-tools/blob/master/build/Scripts/Windows/Build.ps1#L153. Really, I should change this to be [Nn]u[Gg]et.exe